run_game_local.py

```diff
@@ -20,22 +22,23 @@ def run_game(white, black):
 			black.play_move(board)
 		turn += 1
 	else:
-		print(board)
-		# When loop terminates by loop condition, 
+		# When loop terminates by loop condition,
 		# the previous player who moved wins.
 		if board.is_checkmate():
 			if turn % 2 == 0:
-				return "BLACK ({})".format(black.get_name())
+				return "BLACK ({})".format(black.get_name()), board
 			else:
-				return "WHITE ({})".format(white.get_name())
+				return "WHITE ({})".format(white.get_name()), board
 		else:
-			return "DRAW"
+			return "DRAW", board
 
 
 if __name__ == '__main__':
 	print("Starting chess match between two bots")
 	# Replace below RandomBot instantiations with custom Bot implementations
 	white = RandomBot("Random Bot 1")
 	black = RandomBot("Random Bot 2")
-	winner = run_game(white, black)
-	print("RESULT: " + winner)
\ No newline at end of file
+	winner, board = run_game(white, black)
+	#Using chess libray graphics interface to render board
+	chess.svg.board(board=board)
```

Comment:
Can you add an option that lets you specify which output type (graphical or text based) to run the game with.